server/models/handlers/availability.js

```diff
@@ -1,29 +1,24 @@
 // Gets the time availabilities for a specific worker for a specific school.
-async function getWorkerAvailability(workerId, schoolId, startTime, endTime) {
-    const sampleAvailableTimes = [
-        {
-            workerTimeslotId: 1,
-            startTime: '08:00:00', 
-            endTime: '08:30:00',
-            date: '2020-10-20'
-        },
-        {
-            workerTimeslotId: 2,
-            startTime: '08:30:00', 
-            endTime: '09:00:00',
-            date: '2020-10-20'
-        },
-        {
-            workerTimeslotId: 3,
-            startTime: '09:00:00', 
-            endTime: '09:30:00',
-            date: '2020-10-20'
-        }
-    ];
+const _ = require('lodash');
+const availabilityHandler = require('../db/availability');
+//need userTypes or Workerstatus?
+const UserTypes = require('../../constants/userTypes.json');
+
+//
+async function getAvailabilityDetails(workerId, schoolId, startTime, endTime)  {
+   
+    // Don't need this becuase the db will return an error if the worker credentials are invalid.Searches the worker table to see if it is valid worker credentials
+    //const worker = await availabilityModel.getWorker(email, password);
+    if  (_.isNil(workerId)||_.isNil(schoolId)) {
```

Comment:
Hi Nancy,
I would suggest right clicking on this file in Visual Studio and using the Format feature to format the code for better code readability.

Reply:
Hi Amy,
Thank you for suggesting that!
My action: I've reformatted the server/models/handlers/availability.js

Please confirm if this takes care of your suggestion!